### DISCUSSION TITLE
请问有在MacBook Air M1 8GB版上部署7B模型的吗？

### DISCUSSION THREAD

**Discussion by jinleiviva** [ymcui/Chinese-LLaMA-Alpaca-2#154] [General]
请问有在MacBook Air M1 8GB版上部署7B模型的同学吗？我部署了以后，用的llamachat，基本上就是答非所问，不知道是内存不够导致的问题，还是我合并模型过程中出了问题。

**Comment by ymcui** [comment]
llamachat并不支持最新版的llama.cpp格式，也不支持Llama-2-chat的指令模板，和你内存够不够没有关系，不够你都加载不起来。
你需要考虑用其他方式做推理。
另外，虽然不想反复说明，但还是要提醒一下：
1）对话请用Alpaca而不是LLaMA；
2）对话模型大多数都有特定的指令模板，建议你仔细看一下我们的相关wiki。

**Reply by jinleiviva** [reply]
谢谢ymcui回复，我是一个初学者，非常感谢你的耐心指点！这就调整部署策略，感谢你的耐心回复。
